Misc/NEWS.d/next/Core_and_Builtins/2024-12-23-19-24-19.gh-issue-128195.WqaBqL.rst

```diff
@@ -0,0 +1 @@
+Add `_REPLACE_WITH_TRUE` to the tier2 optimizer
```

Comment:
Please remove the news entry. This is internal only so we don't need it. Thanks!

Reply:
👍 Done